### PR TITLE
Add animation property based on React Native Modal config.

### PR DESCRIPTION
**Pull request by lucianomlima**
React Native Modal component [animationType](https://facebook.github.io/react-native/docs/modal.html#animationtype) prop.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -122,9 +125,7 @@ export default class Spinner extends React.Component {
     const { visible } = this.state;
 
     if (!visible)
-      return (
-        <View display="none" />
-      );
```
Comment (niftylettuce) [inline src/index.js]: Why are you returning `null` here?

**Comment by lucianomlima** [comment]
Because `return <View />`, `return <View display={'none'} />` or `return null` has the same effect.
For those who read source code, null is better to understand what is happening.

Isso can revert this line if you think is better.

**Comment by niftylettuce** [comment]
@lucianomlima I think it may need to be reverted to support older versions, but I am unsure.  Also I personally never `return null` as I feel that is bad practice just to write a return value for readability.  A good coder should know and infer that it is not returning anything.

**Comment by lucianomlima** [comment]
@niftylettuce About older versions, the display prop is available only from version 0.43.
Version [0.42] (http://facebook.github.io/react-native/releases/0.42/docs/layout-props.html#display) and earlier do not have a display prop.

**Comment by niftylettuce** [comment]
@lucianomlima I have published v0.5.2 to NPM.  Please install and let me know if it works for you.  Thank you for your contributions.

**Comment by lucianomlima** [comment]
@niftylettuce thanks for merge. I'll check today and comment here.

**Comment by lucianomlima** [comment]
Sorry for the delay, it's working well.

**Pull request merged**